### ISSUE TITLE
Find all objects with a value in a specific annotation key

### ISSUE THREAD

**Issue by o-o00o-o** [otykier/TabularEditor#248]
Using the find in the UI, how can I find all objects with an annotation X that has a value Y in?

I tried looking here https://github.com/otykier/TabularEditor/wiki/Advanced-Filtering-of-the-Explorer-Tree

:GetAnnotation("VR_DataOwner") != null 

works and returns only those objects that have that annotation

but

:GetAnnotation("VR_DataOwner").Contains("Whole") 

doesn't return anything even when I know some of them have this. This is in List mode which I expected this would work. It does work if the annotation is on a table and I use "by parent" mode but it doesn't seem to work if it is

**Comment by otykier** [comment]
I'm missing a null check in the code. As a work-around, until I get it fixed, could you please try to use:

```
:GetAnnotation("VR_DataOwner") != null && GetAnnotation("VR_DataOwner").Contains("Whole")
```

**Comment by otykier** [comment]
Fixed in [2.8.1](https://github.com/otykier/TabularEditor/releases/tag/2.8.1)
